tests/jest/unit/assets/dev/js/editor/document/hooks/ui/settings/set-direction-mode.test.js

```diff
@@ -19,8 +19,6 @@ describe( 'set-direction-mode--document/elements/settings', () => {
 			$previewContents: [ document ],
 		};
 
-		global.CustomEvent = class {};
```

Comment:
IDK why it was here, it's unnecessary and caused issues with the new `dispatchEvent()` of the commands